_posts/2020-09-04-homebrew-2.5.0.md

```diff
@@ -0,0 +1,38 @@
+---
+title: 2.5.0
+author: MikeMcQuaid
+---
+Today I'd like to announce Homebrew 2.5.0. The most significant changes since 2.4.0 are better `brew cask` integration, license support and API deprecations.
+
+Major changes and deprecations since 2.4.0:
+
+- Many `brew cask` [commands](https://github.com/Homebrew/brew/pull/8387) [have](https://github.com/Homebrew/brew/pull/8302) - [been](https://github.com/Homebrew/brew/pull/8387) (@MLH-Fellowship)
+ replaced with `brew` commands and are deprecated. This continues our goal of better integration between `brew` and `brew cask` commands.
+- [Most Homebrew/homebrew-core formulae have their licenses stored and audited](https://github.com/Homebrew/brew/pull/7762).
+- [`brew typecheck` provides the beginnings of Homebrew's use of Sorbet for type-checking](https://github.com/Homebrew/brew/pull/8289).
+- [Homebrew is currently experimenting with GitHub Discussions as a possible replacement for Discourse](https://github.com/Homebrew/brew/pull/8570).
+- [`brew livecheck` has moved from its own tap to a Homebrew command and part of Homebrew/homebrew-core formulae](https://github.com/Homebrew/brew/pull/8180).
+- [`brew bump` is a new command using `brew livecheck` and Repology output to outdated formualae in Homebrew/homebrew-core](https://github.com/Homebrew/brew/pull/7834).
```

Comment:
There's a missing word here between "output to" and "outdated formulae", but I'm bleary eyed so I don't know what it should be. Maybe "check for"?

Reply:
```suggestion
- [`brew bump` is a new command using `brew livecheck` and Repology to output outdated formualae in Homebrew/homebrew-core](https://github.com/Homebrew/brew/pull/7834).
```